### ISSUE TITLE
Crashes server on launch

### ISSUE THREAD

**Issue by OreoTheAvali** [sleepingdragoninn/antique-atlas#14]
Mod Version: 1.1.1 [Beta] Quilt

_Issue:_ Server crashes on launch with antique atlas 4 installed
_Expected:_ Server to not crash on launch

_Reproduction:_
1. Run quilt server with only `qfapi-7.4.0_qsl-6.1.2_fapi-0.90.0_mc-1.20.1` installed on a new world. Observe no crash.
2. Add `antique-atlas-1.1.1+1.20` to server launch with new world. Observe crash.

https://mclo.gs/3XKlMtn

**Comment by xR4YM0ND** [comment]
Same here on Fabric 1.20.1
```js
Cannot load class net.fabricmc.fabric.api.client.networking.v1.ClientPlayConnectionEvents in environment type SERVER
```
[latest.log](https://gist.github.com/xR4YM0ND/9a58a442cd3b1c9450293e0244786ec7)

**Comment by sleepingdragoninn** [comment]
should be fixed in 1.1.2 - thanks for the report!
